src/Tickets/Order_Modifiers/Custom_Tables/Order_Modifier_Relationships.php

```diff
@@ -64,21 +61,16 @@ class Order_Modifier_Relationships extends Table {
 	 */
 	protected function get_definition() {
 		global $wpdb;
-		$table_name        = self::table_name( true );
-		$charset_collate   = $wpdb->get_charset_collate();
-		$parent_table_name = Order_Modifiers::table_name();
-		$parent_table_uid  = Order_Modifiers::uid_column();
-		$wp_posts_table    = $wpdb->posts;
+		$table_name      = self::table_name( true );
+		$charset_collate = $wpdb->get_charset_collate();
 
 		return "
 			CREATE TABLE `$table_name` (
 				`object_id` BIGINT UNSIGNED NOT NULL AUTO_INCREMENT,
 				`modifier_id` BIGINT UNSIGNED NOT NULL,
 				`post_id`  BIGINT UNSIGNED NOT NULL,
 				`post_type` VARCHAR(20) NOT NULL,
-				PRIMARY KEY (`object_id`),
-				FOREIGN KEY (`modifier_id`) REFERENCES $parent_table_name($parent_table_uid)ON DELETE CASCADE,
-    			CONSTRAINT `fk_post_id` FOREIGN KEY (`post_id`) REFERENCES `$wp_posts_table` (`ID`) ON DELETE CASCADE
```

Comment:
Its ok to create keys for indexing so things are faster, but dont create actual MySQL references to entities, it causes lots of issues.